docs/src/man/conclusion.md

```diff
@@ -98,7 +98,7 @@ mutable struct CSVFormatter <: Formatter
     delim::Char
     vals::Array{Symbol}
 
-    CSVFormatter(delim=',', vals=Array{Symbol}()) = new(delim, vals)
+    CSVFormatter(delim=',', vals=Array{Symbol}(undef)) = new(delim, vals)
```

Comment:
bad bot

Reply:
good bot

Reply:
At least in that the previous definition was wrong. The new definition does something differently-wrong. Should be `Vector{Symbol}()`.